Backend/app/rider-platform/rider-app/Main/src/API/UI/Confirm.hs

```diff
@@ -62,7 +63,7 @@ confirm ::
   FlowHandler ConfirmRes
 confirm personId quoteId =
   withFlowHandlerAPI . withPersonIdLogTag personId $ do
-    dConfirmRes <- DConfirm.confirm personId quoteId
+    dConfirmRes <- SConfirm.confirm personId quoteId
```

Comment:
Don't directly import from ShareLogic here. Instead of that just create confirm function inside Action/UI and then internally in that function call SharedLogic confirm function.